cs/ccxt/base/Throttler.cs

```diff
@@ -127,6 +126,10 @@ private async Task rollingWindowLoop()
             }
             else
             {
+                if (timestamps.Count <= 0)
+                {
+                    continue;
+                }
```

Comment:
@caoilainnl this?

Reply:
one line below this, we access `timestamps[0]`, and at first it's good to check if the list length is 0, then just skip